### PR TITLE
Restore stakeholder and personas 

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
- Updates stakeholders and personas doc in research syntheses subfolder to reflect recent changes (adds OCIO staff role, removes outdated next steps, removes outdated commentary on TDP releases)
- Updated UX readme to link directly to the doc for improved visibility
- Updated Research Syntheses readme with a non placeholder description for the doc

### CODE REVIEW

**Comment by lfrohlich** [comment]
as @ADPennington  pointed out, this already exists here: https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research-Syntheses/2020%2C%20Summer%20-%20Understanding%20Stakeholders%20and%20creating%20personas.md. Per our discussion 3/14, please add a high level link to this summary in the User Experience readme

**Comment by ADPennington** [comment]
@reitermb could you also fill-in the  missing blurb in the synthesis readme re: personas (https://github.com/raft-tech/TANF-app/tree/raft-tdp-main/docs/User-Experience/Research-Syntheses)

**Comment by reitermb** [comment]
@lfrohlich @ADPennington PR should be updated now with the changes discussed in UX Sync!

File: docs/User-Experience/README.md [hunk 1]
```diff
@@ -11,16 +11,17 @@ This directory includes the following research- and design-related docs and reso
...
-  - Support the overall team's alignment on *what* needs to be researched and delivered to support functionality in the product roadmap
+  - Support the overall team's alignment on *what* needs to be researched and delivered to support functionality in the product roadmap.
   - Support the design team in determining research strategy.
   - Support the design team in pre-planning research to reduce the planning lift of prioritizing any given research topic.
 - **[User Research guide](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/User-Research-Guide.md)** - covers practices and considerations for ensuring participant privacy and safety (including psychological safety).
-- **[Research Syntheses](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research-Syntheses)** - Folder for storing findings from user research sessions
+- **[Research Syntheses](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research-Syntheses)** - Folder for storing findings from user research sessions.
```
Comment (ADPennington) [inline docs/User-Experience/README.md:18]:
```suggestion
- **[Research Syntheses](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research-Syntheses)** - Folder for storing findings from user research sessions. This includes an overview of **_[Stakeholder types and Personas](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research-Syntheses/2020%2C%20Summer%20-%20Understanding%20Stakeholders%20and%20creating%20personas.md)_** in use on the project.
```

File: docs/User-Experience/README.md [hunk 2]
```diff
@@ -11,16 +11,17 @@ This directory includes the following research- and design-related docs and reso
...
+- **[Stakeholders and Personas](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Research-Syntheses/2020%2C%20Summer%20-%20Understanding%20Stakeholders%20and%20creating%20personas.md)** - Overviews project stakeholder types and personas in use on the project.
```
Comment (ADPennington) [inline docs/User-Experience/README.md:19]:
```suggestion
```

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 3]
```diff
@@ -105,9 +100,10 @@ Regional program specialists are grantee's primary OFA point of contact. They tr
 
 The Office of Planning, Research, and Evaluation (OPRE) studies ACF programs and the populations they serve through rigorous research and evaluation projects. They use TANF data in their analyses. More research is needed in this area, especially on where duties and responsibilities overlap with OFA analysts.
 
-### OCIO staff
+### OCIO Staff
```
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md:103]:
```suggestion
### ACF OCIO Staff
```

**Review by ADPennington — APPROVED** [review]
lgtm @reitermb 👍🏾

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 4]
```diff
@@ -12,22 +10,19 @@ Last updated for [Issue #804](https://github.com/raft-tech/TANF-app/issues/804)
...
 
 We expect that there will be a couple hundred users who interact with the TDP system, made up of both STT and OFA staff.
 
-For the purposes of the Release 1, the only users who will have access to the application will be OFA Admin and STT (test) users.  Raft team or OFA will be acting as STT users for the OFA MVP.
-
 **Supporting Documentation**
 
 - [Personas mural](https://app.mural.co/t/officeoffamilyassistance2744/m/gsa6/1592254280716/2ae8293a3233a95941d548cda4f373faab96b40b) :lock:
-- [Vision and Stakeholders Doc](https://github.com/HHS/TANF-app/blob/main/docs/Product-Strategy/Vision-and-Stakeholders.md)
+- [Vision and Stakeholders Doc](https://github.com/raft-tech/TANF-app/blob/restore-stakeholder-and-personas/docs/Product-Strategy/Vision-and-Stakeholders.md)
```
[the remark this answers is not in the document]
Comment (reitermb) [reply docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md:25]: It's a working link—I noticed some inconsistency in whether we're linking to the HHS repo or the raft-tech one so I defaulted to the norm in the UX folder which seemed to be to use raft-tech links. Possibly something to standardize / fix en masse at some point if that's not the paradigm we want?

**Pull request merged**